pandablocks/utils.py

```diff
@@ -0,0 +1,141 @@
+from typing import Dict, List, Sequence, Union
+
+import numpy as np
+import numpy.typing as npt
+
+from pandablocks.responses import TableFieldInfo
+
+UnpackedArray = Union[
+    npt.NDArray[np.int32], npt.NDArray[np.uint8], npt.NDArray[np.uint16]
+]
+
+
+def words_to_table(
+    words: Sequence[str], table_field_info: TableFieldInfo
+) -> Dict[str, List]:
```

Comment:
I get errors when trying to use Sequence (apparently numpy arrays aren't considered sequences because they don't have `.count` and `.index` functions).